cfgov/jinja2/v1/learn-page/index.html

```diff
@@ -2,12 +2,18 @@
 
 {# Import organisms and molecules used in the template. #}
 
+
 {% import 'molecules/related-links.html' as related_links %}
 {% import 'molecules/quote.html' as quote %}
+
 {% import 'organisms/full-width-text.html' as full_width_text %}
+{% import 'organisms/sidebar-contact-info.html' as sidebar_contact_info %}
 {% import 'organisms/post-preview.html' as post_preview %}
 {% import 'organisms/sidebar-contact-info.html' as sidebar_contact_info %}
 
+{% import 'atoms/large_checkbox.html' as large_checkbox %}
+{% import 'atoms/large_radio.html' as large_radio %}
```

Comment:
Technically the filenames should be `large-checkbox.html` and `large-radio.html` to follow the format used for the rest of the includes. Converting the existing files in `/organisms/*` etc to snake case as well is an option, but they should be consistent.